lib/components/app/desktop-nav.tsx

```diff
@@ -104,16 +105,21 @@ const DesktopNav = ({
         >
           <Navbar.Brand>
             <AppMenu />
-            {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
-            {/* @ts-ignore The dynamic tag is causing some trouble */}
-            <BrandingElement
-              className={branding && `with-icon icon-${branding}`}
-              {...brandingProps}
-            >
-              {/* A title is always rendered (e.g.for screen readers)
-                  but is visually-hidden if a branding icon is used. */}
-              <div className="navbar-title">{title}</div>
-            </BrandingElement>
+
+            {!isMobile() && (
```

Comment:
This removes the logo while in the account page. Is that the desired effect?

Reply:
Yes. The logo is not shown in the nav bar anywhere on the site while on mobile. To maintain consistency we will need to add the H1 tag in the navbar later down the road. The TODO comment in app-frame.tsx is where that logic will need to go.